src/data/code-samples/en/accessibility-focus/swift-ui.md

```diff
@@ -0,0 +1,16 @@
+# Accessibility focus - SwiftUI
+
+In SwiftUI, enhancing the accessibility of your app can be achieved by using the [accessibilityFocused(_:)](https://developer.apple.com/documentation/swiftui/view/accessibilityfocused(_:)) view modifier. This modifier allows you to bind an element's focus state to a boolean variable, enabling you to track and respond to focus changes in a dynamic and visually informative way.
```

Comment:
Suggestion for 1st sentence: 'In SwiftUI, you can move accessibility focus using the accessibilityFocused modifier"